humanguide/other-design-considerations/convergence.rst

```diff
@@ -0,0 +1,22 @@
+Convergence
+===========
+
+.. figure:: /_static/images/humanguide/41.png
+
+Convergence is one of the milestones Ubuntu Touch wants to achieve. One app available across all screen sizes and input methods, such as mobile handsets, tablets, and laptops.
+
+Screen size
+-----------
+
+.. figure:: /_static/images/humanguide/phone-tablet-UTTT.png
+
+   Ubuntu Touch Tweak Tool app
+
+Main page stays as the main page on phones or on the left on bigger screen devices. Additional pages on the phone, can be displayed at variable sizes on the right on bigger screens.
```

Comment:
"Main page stays as main page" is unclear. Are you trying to convey that the main view is transitioned to the child view on small screen sizes?